adaptors/library/jobs/satusehat-create-medication.js

```diff
@@ -0,0 +1,134 @@
+// Create a medication from visit data and lookup table
+
+function generateUUID() {
```

Comment:
Not keen on spreading this pattern! It's not a nice function at all

Common should have a util.uuid. Can we use that instead? If it's not exported by satusehat, let's do a quick patch (just make sure common.util is exported by satusehat)

Reply:
@josephjclark  We are not exporting the `util.uuid` from `common` in satusehat

Reply:
Then let's do a patch and do that :) Should only take a minute an it'll make this example significantly cleaner

I've only had a light look over this btw - I'll get back to you on the rest of it later